### PR TITLE
chore: prepare @griffel/webpack-extraction-plugin for publish

### PR DESCRIPTION
**Pull request by layershifter**
This PR:
- adds proper `README` file to `@griffel/webpack-extraction-plugin`
- removes `private: true` from its `package.json` to publish the package

### CODE REVIEW

File: packages/webpack-extraction-plugin/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@griffel/webpack-extraction-plugin",
   "version": "0.0.1",
-  "private": true,
```
Comment (layershifter) [inline packages/webpack-extraction-plugin/package.json:4]: The plan is to publish it as `0.0.x` as I don't have enough confidence in it.

File: packages/webpack-extraction-plugin/README.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+> ⚠️ **Note** This package is experimental, API may change.
+
+A plugin for Webpack 5 that performs CSS extraction for [`@griffel/react`](../react).
+
+## Install
+
+```bash
+yarn add --dev @griffel/webpack-extraction-plugin
+# or
+npm install --dev @griffel/webpack-extraction-plugin
```
Comment (Dwlad90) [inline packages/webpack-extraction-plugin/README.md:12]: Hi, I guess that is the right way to install the devDependencies: 

```suggestion
npm install --save-dev @griffel/webpack-extraction-plugin
```

File: packages/webpack-extraction-plugin/README.md [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+```js
+const { GriffelCSSExtractionPlugin } = require('@griffel/webpack-extraction-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(js|ts|tsx)$/,
+        // Apply "exclude" only if your dependencies **do not use** Griffel
```
Comment (ling1726) [inline packages/webpack-extraction-plugin/README.md:46]: should `include` be added to the docs too?

File: packages/webpack-extraction-plugin/README.md [hunk 4]
```diff
@@ -1,3 +1,71 @@
...
+      },
+      // Add "@griffel/webpack-loader" if you use Griffel directly in your project
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: '@griffel/webpack-loader',
+        },
+      },
+      // ".css" configured there for example, but "css-loader" is required to handle produces CSS assets
```
Comment (ling1726) [inline packages/webpack-extraction-plugin/README.md:60]: >.css configured there for example

what does this mean ?

File: packages/webpack-extraction-plugin/README.md [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+    ],
+  },
+  plugins: [new MiniCssExtractPlugin(), new GriffelCSSExtractionPlugin()],
+};
+```
+
+- Usage `mini-css-extract-plugin` is not required
```
Comment (ling1726) [inline packages/webpack-extraction-plugin/README.md:71]:
```suggestion
- Usage `mini-css-extract-plugin` is not required
```

Can you clarify this point? Do you mean that the css will simply be in the main js bundle if mini-css-extract-plugin is not used ?

File: packages/webpack-extraction-plugin/README.md (hunk 3 above)
Comment (layershifter) [reply packages/webpack-extraction-plugin/README.md:47]: Added a section below 64a52cd

File: packages/webpack-extraction-plugin/README.md (hunk 4 above)
Comment (layershifter) [reply packages/webpack-extraction-plugin/README.md:61]:
```suggestion
      // "css-loader" is required to handle produced CSS assets by Griffel
      // you can use "style-loader" or "MiniCssExtractPlugin.loader" to handle CSS insertion
```

Is it better?

File: packages/webpack-extraction-plugin/README.md (hunk 5 above)
Comment (layershifter) [reply packages/webpack-extraction-plugin/README.md:72]:
```suggestion
- `mini-css-extract-plugin` is not mandatory and configured there for example, you can use `style-loader` or other tooling to inject CSS on a page
```

Is it better?

Comment (ling1726) [reply packages/webpack-extraction-plugin/README.md:72]: yup that's much better

File: packages/webpack-extraction-plugin/README.md (hunk 4 above)
Comment (ling1726) [reply packages/webpack-extraction-plugin/README.md:61]: yup looks good

**Review by ling1726 — APPROVED** [review]

**Pull request merged**